Makefile

```diff
@@ -11,7 +11,7 @@ setup:
 
 .PHONY: run
 run: setup
-	yarn start
+	NODE_OPTIONS=--openssl-legacy-provider yarn start
```

Comment:
For the moment, this is a workaround. Without that flag, OpenSSL has errors. On the one hand, we don't really need harder OpenSSL security defaults (I think), because we have no backend. However, introducing workarounds for legacy options is probably also not the best idea.

Reply:
I think this has been introduced since node v17. So, if we cannot fix it and it is a hard problem, we could try upgrading to v16 first. However, v16 is not maintained anymore, as well. So that's not the greatest of all options..